kitsune/search/base.py

```diff
@@ -117,16 +122,6 @@ def alias_points_at(cls, alias):
 
         return aliased_indices[0] if aliased_indices else None
 
-    @classmethod
-    @property
-    def update_document(cls):
```

Comment:
This change will make the index_object to fail. If I recall correctly it operates on class types and not on instantiated objects.